index.html

```diff
@@ -65,67 +65,10 @@ <h3>Multi-Post Stories</h3>
                 </div>
             </div>
             <div class="works-container" id="works-container">
```

Comment:
- [ ] You did great implementing the details pop. :+1: :100: Your pop-up works great but I think there are some styling issues you need to improve on before moving to the next milestone. 
- Looking at the Fimga design, when the pop-up is open, the background is blurred, which helps keep the pop-up window in focus, As part of the requirement for this milestone is to replicate the given design as much as you can, I'll suggest you take a look at this and add the blur background styling. :waffle: :smile: 
- Also following the given design, the Project image, takes more width than the content, I'll suggest you adjust their width to accurately follow the given design. :heavy_check_mark: :smile: 

- - Your Current design:
![Selection_041](https://github.com/Luffytaro22/Portfolio-Mobile-First-Skeleton/assets/107632780/8591278e-727a-432e-b746-073a5fd0b026)

- - Figma design:
![Selection_040](https://github.com/Luffytaro22/Portfolio-Mobile-First-Skeleton/assets/107632780/c7ccdb0b-6a78-4054-8af2-686232b909be)